pkgs/development/tools/godot/3/mono/make-deps.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+nix-shell ../../../../../../ -A godot3-mono.make-deps --run 'eval "$makeDeps"'
```

Comment:
From which folder is this executed? From nixpkgs root? This looks a bit strange...

Reply:
There already was a discussion about this above but no one had an idea on how to do this in a better.

Reply:
This is intended to be run from within the pkgs/development/tools/godot/3/mono folder within nixpkgs.  It will drop a file there.  Couldn't come up with a better way to handle it.  There are other packages with update scripts, but none of them run nix-shell because they just write all the steps for updating directly in the sh script.  In mine, I wanted to actually run some of the phases from the godot derivation at the beginning of the update script, which is why I'm running a nix shell.

Reply:
I know a few update scripts that use `git rev-parse --show-toplevel` to find the top-level dir for nixpkgs. That way you can also run it from any directory in nixpkgs and have it find the right entrypoint

Maybe that would work here?

Reply:
Seems so.  Just tried it out, and that works as a substitute for the `..`s